pennylane_lightning/src/bindings/Bindings.cpp

```diff
@@ -352,20 +355,19 @@ PYBIND11_MODULE(lightning_qubit_ops, // NOLINT: No control over Pybind internals
 
     m.def("generateBitPatterns",
           py::overload_cast<const vector<size_t> &, size_t>(
-              &IndicesUtil::generateBitPatterns),
+              &Gates::generateBitPatterns),
           "Get statevector indices for gate application");
     m.def("getIndicesAfterExclusion",
           py::overload_cast<const vector<size_t> &, size_t>(
-              &IndicesUtil::getIndicesAfterExclusion),
+              &Gates::getIndicesAfterExclusion),
```

Comment:
We can probably look at removing these old functions in a follow-up PR, since (I think) your implementations handle all indexing in-place (or at least much more efficiently).

Reply:
+1 Probably we can.